src/XamarinBackgroundKit.iOS/Renderers/MaterialContentViewRenderer.cs

```diff
@@ -23,8 +24,6 @@ public override void LayoutSubviews()
             BackgroundManager?.InvalidateLayer();
         }
 
-        #region Element Changed
```

Comment:
Is there any reason for removing regions?

Reply:
Sorry, my plugin(CodeMaind) remove regions because is a anti-pattern